near-sdk-core/src/code_generator/trait_item_method_info.rs

```diff
@@ -43,4 +22,28 @@ impl TraitItemMethodInfo {
             }
         }
     }
+
+    pub fn generate_serialier(attr_sig_info: &AttrSigInfo, serializer: &SerializerType) -> TokenStream2 {
+      let has_input_args = attr_sig_info.input_args().next().is_some();
+      if !has_input_args {
+        return quote! {let args = vec![]; }
+      }
```

Comment:
I think for json we need `b"{}".to_vec()`. Hmm, how did it work before? Do we ever have no args?

Reply:
This update assumes that the local `args` will be a byte vector.  So in the case where there are no arguments this is the default value.  Then the other paths for serializing each create an `args` with the same type.